common/utils.js

```diff
@@ -403,10 +403,24 @@ export function matchClassification(a, b) {
 }
 
 /**
- * Reads data from local file.
+ * Reads binary data from local file.
  * 
  * @param {string} fileName 
- * @returns {any}
+ * @returns {Promise<any>}
+ */
+export async function readBinary(fileName) {
+    return new Promise((resolve) => {
+        fs.readFile(fileName, (err, contents) => {
```

Comment:
NodeJS ships sync versions of `fs` APIs. IMO it's better fit here for the sake of code redability,

Reply:
Fixed